### DISCUSSION TITLE
Extra Options for PWA

### DISCUSSION THREAD

**Discussion by Coragyps** [Floorp-Projects/Floorp#893] [Feature Request]
One thing I liked from edge was the option to edit PWA _name_ and _icon_ at the moment of creation (Since sometimes the default favicon has a low resolution).

![exampl](https://github.com/Floorp-Projects/Floorp/assets/24980774/0093d831-f6e3-4dc9-b5f2-5986acfd3f52)

Also, when checking "**Use Titlebar**" on _customize toolbar_ (to enable the native windows titlebars), the navigation bar remains present even if "**Disable toolbars on Web Apps Window**" is enabled. This could be an option for users that only want the titlebar present.

**Comment by CutterKnife** [comment]
> Also, when checking "Use Titlebar" on customize toolbar (to enable the native windows titlebars), the navigation bar remains present even if "Disable toolbars on Web Apps Window" is enabled. This could be an option for users that only want the titlebar present.

This maybe a bug, try opening an issue.
